src/features/pmicControl/npm/npm1300/documentationPmic1300.tsx

```diff
@@ -818,6 +818,20 @@ export const documentation: Documentation = {
                 ],
             },
         ],
+        IBATLIM: [
+            {
+                title: <span>Battery Discharge Current Limit</span>,
+                content: [
+                    <p key="p1">
+                        For best fuel gauge accuracy, use
+                        <span>IBAT</span>
```

Comment:
```suggestion
                        For best fuel gauge accuracy, use
                        {` `}<span>IBAT</span>{` `}
```
We need blank spaces around this span. Check the image in your comment below.